apps/charterafrica/src/payload/collections/People.js

```diff
@@ -0,0 +1,153 @@
+import { PEOPLE_COLLECTION } from "../../lib/tools/models";
+import slug from "../fields/slug";
+import source from "../fields/toolSourceField";
+import updatedAt from "../fields/updatedAt";
+
+const People = {
+  slug: PEOPLE_COLLECTION,
+  labels: {
+    singular: {
+      en: "People",
+      fr: "People",
+      pt: "Pessoas",
+    },
+    plural: {
+      en: "People",
+      fr: "People",
+      pt: "Pessoas",
+    },
+  },
+  admin: {
+    useAsTitle: "externalId",
+  },
+  access: {
+    read: () => true,
+  },
+  fields: [
+    source(),
+    {
+      name: "externalId",
+      type: "text",
+      required: true,
+      label: { en: "External ID", fr: "ID externe", pt: "ID externo" },
+      admin: {
+        readOnly: true,
+      },
+    },
+    {
+      name: "fullName",
+      type: "text",
+      label: {
+        en: "Full Name",
+        fr: "Nom complet de la personne",
+        pt: "Nome completo da Pessoa",
+      },
+      admin: {
+        readOnly: true,
+      },
+    },
+    slug({ fieldToUse: ["source", "username"] }),
+    {
+      name: "username",
+      type: "text",
+      label: {
+        en: "Username",
+        fr: "Nom d'utilisateur de la personne",
+        pt: "Nome de usuário da Pessoa",
+      },
+      admin: {
+        readOnly: true,
+      },
+    },
+    {
+      name: "description",
+      type: "textarea",
+      label: {
+        en: "Description",
+        fr: "Description de la personne",
+        pt: "Descrição da Pessoa",
+      },
+      admin: {
+        readOnly: true,
+      },
+    },
+    {
+      name: "type",
+      type: "text",
+      label: { en: "Type", fr: "Taper", pt: "Tipo" },
+      admin: {
+        readOnly: true,
+      },
+    },
+    {
+      name: "avatarUrl",
+      type: "text",
+      label: { en: "Avatar URL", fr: "URL d'avatar", pt: "URL de avatar" },
+      admin: {
+        readOnly: true,
+      },
+    },
```

Comment:
Not sure if its possible, but can we maybe display the avatar instead of just the URL

Reply:
@kelvinkipruto that's an overkill I think. 
We only need the URL as we get from github